ionic2/app/components/map/map.component.ts

```diff
@@ -34,9 +35,14 @@ export class MapComponent {
     this.map.goTo(position);
   }
 
-  filter(filterOptions: FilterOptions) {
-    console.debug('map:filter', filterOptions);
-    this.map.filter(filterOptions);
+  filter(filter: Filter) {
+    let filterObj:FilterOptions = {
+      pokemonIds: filter.selectedPokemon,
+      sightingsSince: filter.sightingsRange,
+      predictionsUntil: filter.predictionsRange
+    };
+    console.debug('map:filter', filter);
```

Comment:
maybe a `//FIXME` wouldn't hurt here, especially now as the project ends soon, and we might forget things